modules/vf-graphql-holochain/mutations/proposedTo.ts

```diff
@@ -5,26 +5,32 @@
  * @since:   2019-09-12
  */
 
-import { DNAIdMappings } from '../types'
+import { AgentAddress, ByRevision, DNAIdMappings, ProposalAddress } from '../types'
 import { mapZomeFn } from '../connection'
 import { deleteHandler } from './'
 
 import {
   ProposedToResponse,
 } from '@valueflows/vf-graphql'
 
+export interface CreateArgs {
+  proposedTo: {
+    proposed: ProposalAddress,
+    proposedTo: AgentAddress,
+  },
+}
 export type createHandler = (root: any, args) => Promise<ProposedToResponse>
 
 export default (dnaConfig: DNAIdMappings, conductorUri: string) => {
-  const runCreate = mapZomeFn(dnaConfig, conductorUri, 'proposal', 'proposed_to', 'create_proposed_to')
-  const runDelete = mapZomeFn(dnaConfig, conductorUri, 'proposal', 'proposed_to', 'delete_proposed_to')
+  const runCreate = mapZomeFn<CreateArgs, ProposedToResponse>(dnaConfig, conductorUri, 'proposal', 'proposed_to', 'create_proposed_to')
+  const runDelete = mapZomeFn<ByRevision, boolean>(dnaConfig, conductorUri, 'proposal', 'proposed_to', 'delete_proposed_to')
 
   const proposeTo: createHandler = async (root, args) => {
-    return runCreate({ proposed_to: args })
+    return runCreate({ proposedTo: args })
```

Comment:
```suggestion
    return runCreate(args)
```
I think without reverting, this will be a bug. Add the typescript definition that's missing for `args` to the above `createHandler` type definition.

Reply:
I retract, thanks for the clarification. In the future it would be good to clean this up. I've heard these proposal things are somewhat relatively new, but they broke from the convention a bit so they look out of place.